src/commands/containers/restartContainer.ts

```diff
@@ -18,7 +18,7 @@ export async function restartContainer(context: IActionContext, node: ContainerT
 
     await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: "Restarting Container(s)..." }, async () => {
         await Promise.all(nodes.map(async n => {
-            await ext.dockerode.getContainer(n.container.Id).restart();
+            await n.getContainer().restart();
```

Comment:
Fair enough - I'll try to avoid it (I usually only do it for small arrow functions in things like `map` or `sort`). This case is a little unique because there's already a "node" variable defined outside this scope which would cause conflicts.